relay/pkg/btc/remote/remote.go

```diff
@@ -97,9 +97,9 @@ func (rc *remoteChain) GetHeaderByHeight(height *big.Int) (*btc.Header, error) {
 	}
```

Comment:
I have already noted it and I know I am annoying but `btc/remote/remote.go` and `btc/local/local.go` feels wrong to me especially that they are exported. We can, for example, do `btc/client.go` and `btc/local_client.go` and have two functions, `Connect()` and `ConnectLocal()`

Reply:
Not in this PR of course, just grumbling. 😁

Reply:
Yeah, that's probably a good move. I'll leave a TODO and address this in a follow-up PR.

Reply:
Follow-up PR: https://github.com/keep-network/tbtc/pull/770